### ISSUE TITLE
Parent galaxy should not be part of comparison

### ISSUE THREAD

**Issue by PBarmby** [mshubat/galaxy_data_mines#6]
Of course, solving #1 creates a new issue - which is that querying by coordinates leads to the galaxy under consideration being part of the output tables. And it's not really relevant to what we're interested in. Working on dealing with by modifying the `reformat_table` function.

**Comment by PBarmby** [comment]
Agreed, the previous implementation of query by name is likely fine in most circumstances. It comes up more as an issue for smaller and/or more obscure galaxies, not big things like the Messier objects.

**Comment by PBarmby** [comment]
OK, so I looked into this some more and it's not as easy to fix as I thought. NED and SIMBAD do not always return the same coordinates for a given galaxy and sometimes the differences are substantial (eg for nearby dwarf galaxy IC10, the positions differ by a full arcmin). This means that you can't just match by coordinates, and because NED & SIMBAD's naming schemes are just different enough to be annoying, you can't automatically match by name either. I don't really see how to deal with this, so ignoring it for now.

**Comment by mshubat** [comment]
Ya I can see that being more difficult to deal with. Kind of makes me wanna just do searches directly bycoordinates rather than fussing with the 2 different name resolution results. 

Alternatively maybe we could do a check for differences in location of querybyname coordinates and if they are beyond a certain threshold then we could throw a coordinate mismatch exception or something...

Another idea which just occurred to me is using the is using the -obj-radius option and making that larger to match the disparates in the coordinates for a given name resolution.

Other than that its kind of hard to deal with unless we start doing the name resolution to coordinates ourselves (which is probably a lot of work), or find another system that already does this for us.
